### ISSUE TITLE
Clarify the dependencies to use with the generated tests

### ISSUE THREAD

**Issue by MaelAudren** [STAMP-project/evosuite-model-seeding-tutorial#6]
#### Characteristics
- **Issue Type**: [test report]
- **Reproducibility**: [always]
- **Severity**: [major]
- **Reporter**: [Mael AUDREN DE KERDREL, mael.audren@activeeon.com]

#### Description

I would like to know the right dependencies to add in order to avoid conflict in the future and if evosuite-master:1.0.6 is the good one.

Including the generated test in a project require to include new dependencies because tests are created using evosuite libraries.
I had issues to find the good dependency to add a by adding to much dependencies I had conflict between evosuite dependency with the same version.

I end up adding the following line in the build.gradle:
`testCompile group: 'org.evosuite', name: 'evosuite-master', version: '1.0.6'`.

I discover that this dependency has a conflict with the dependencies:
```
testCompile group: 'org.evosuite', name: 'evosuite-runtime', version: '1.0.6'
testCompile group: 'org.evosuite', name: 'evosuite-client', version: '1.0.6'
```
Using those dependencies prevent to compile the mock method that is used in generated tests.

#### Steps to reproduce

```
git clone https://github.com/ow2-proactive/catalog
git clone https://github.com/STAMP-project/evosuite-model-seeding-usecases-output
cp evosuite-model-seeding-usecases-output/activeeon/catalog/evosuite-tests/org/ow2/proactive/catalog/rest/controller/BucketController_ESTest.java catalog/src/test/java/org/ow2/proactive/catalog/rest/controller/
cd catalog
./gradlew build
```

**Comment by xdevroey** [comment]
Hi @MaelAudren, the dependency to import is evosuite-standalone-runtime (see [How to run the generated JUnit tests](http://www.evosuite.org/documentation/commandline/) in the EvoSuite documentation). I will add this information to the tutorial. Thank you for reporting.

Just one question: I do not understand where the two EvoSuite dependencies (client and runtime) come from. Is it something that you added in your Gradle file?

**Comment by MaelAudren** [comment]
I look at the missing dependencies in the generated class then I search on maven repository which jar could solve the dependencies using the package name as a clue.
Here are the dependencies that were missing in the class that I analysed:
- org.evosuite.shaded.org.mockito.Mockito.*
- org.evosuite.runtime.ViolatedAssumptionAnswer
- org.evosuite.runtime.javaee.injection.Injector

Indeed I added the dependencies evosuite-runtime and evosuite-client in the Gradle file because they solved the dependencies. Then I realize that they were in conflict with evosuite-master jar that solves the same dependencies so I remove the unnecessary jars.
